routers/routes/chi.go

```diff
@@ -195,6 +196,8 @@ func NewChi() chi.Router {
 	if setting.EnableAccessLog {
 		setupAccessLogger(c)
 	}
+	c.Use(middleware.Timeout(60 * time.Second))
```

Comment:
This should be moved out as constant or better as setting imho

Reply:
Maybe make it 2 minutes default, configurable.

Reply:
I just removed the middleware because I don't know what's the default timeout is suitable.